frontend/src/app/scripts/_components/ScriptItems/DefaultSettings.tsx

```diff
@@ -1,35 +1,53 @@
 import { Script } from "@/lib/types";
 
 export default function DefaultSettings({ item }: { item: Script }) {
-  const hasAlpineScript = item?.expand?.alpine_script !== undefined;
+  const defaultSettings = item.install_methods.find(
+    (method) => method.type === "default",
+  );
+
+  const defaultSettingsAvailable =
+    defaultSettings?.resources.cpu !== null ||
```

Comment:
```suggestion
    defaultSettings?.resources.cpu ||
```
to handle empty string and undefined if `defaultSettings` is not set